src/platform/test/functional/apps/dashboard/group6/dashboard_snapshots.ts

```diff
@@ -92,7 +92,7 @@ export default function ({
       );
 
       await dashboard.clickExitFullScreenLogoButton();
-      expect(percentDifference).to.be.lessThan(0.029);
+      expect(percentDifference).to.be.lessThan(0.29);
```

Comment:
Temporarily updated the value here, but I'd like to update the baseline.

Existing baseline:

![es70-ftr-area_chart-01-before](https://github.com/user-attachments/assets/16400d9d-9672-4390-af20-c9be9c2ed6dc)

This PR:

![es70-ftr-area_chart-02-after](https://github.com/user-attachments/assets/22659cfe-8f4d-41ec-a8ae-1c6d1a43b53a)